tests/jballerina-unit-test/src/test/resources/test-src/expressions.typecast.foo/Ballerina.toml

```diff
@@ -1,3 +1,4 @@
-[project]
-org-name = "ballerina-test"
+[package]
+org = "test_org"
+name = "typecast"
 version = "0.0.0"
```

Comment:
Shall we add a new line here ?

Reply:
+1. Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/26802/commits/f6aa23950cf37c29926c3e48d1cc2492591fb017